lib/plugins/features/movies/controller.js

```diff
@@ -3,7 +3,14 @@
 const Movie = require('../../../models/movie');
 
 exports.create = (payload) => {
-  return new Movie().save(payload)
+  const payloadCopy = Object.assign({}, payload);
+
+  if (payloadCopy.hasOwnProperty('title')) {
+    payloadCopy.name = payloadCopy.title;
```

Comment:
I just realized there's definitely a way to do this in the validator through JOI. I'm not sure how that is done so don't worry about it here, but just so that you know. It's nice to keep as much of the payload manipulation in the validator as possible.